package.json

```diff
@@ -66,6 +66,7 @@
     "@jsdevtools/ono": "^7.1.3",
     "@readme/better-ajv-errors": "^1.6.0",
     "@readme/json-schema-ref-parser": "^1.2.0",
+    "@readme/openapi-schemas": "^3.0.1",
```

Comment:
can you install `@apidevtools/openapi-schemas`?

Reply:
uninstalled!